facade/pkg/check/check.go

```diff
@@ -0,0 +1,28 @@
+package check
+
+import (
+	"fmt"
+	"strings"
+)
+
+//CheckFormat struct
```

Comment:
https://stackoverflow.com/questions/30109674/documenting-public-methods-on-private-structs-via-godoc-in-go
https://www.digitalocean.com/community/tutorials/how-to-write-comments-in-go

Reply:
done https://github.com/santa-freddy/go_progress/commit/14cd9401438926e383dc8aeb1cefb3b8223f1584